vars/main.yml

```diff
@@ -32,3 +32,22 @@ rhel7stig_re_qp_key_end: (?:" *)
 
 # insert the parameter at the beginning or append to the end, default append
 rhel7stig_re_qp_insert: "{{ insert | default(not (append | default(true))) }}"
+
+#Developers said the STIG lockdowns made ssh too difficult as the CAT2 finding says we must disconnect after 10 minutes of inactivity on ssh and console sessions. This will not be done on production/integration environments, but if teams want it in their sandbox, modify the following values:
+# rhel_07_040320_ssh_keepalive_interval:
+# rhel_07_040160_console_keepalive_interval:
+#Both values are in seconds (60 seconds * 10 = 600)
+             
+#The following will set the ssh timeout to 1 day:
+#RHEL-07-040320 | All network connections associated with SSH traffic must terminate at the end of the session or after 10 minutes of inactivity, except to fulfill documented and validated mission requirements
+rhel_07_040320_ssh_keepalive_interval: 28800
+             
+#RHEL-07-040160 | All network connections associated with a communication session must be terminated at the  the session or after 10 minutes of inactivity from the user at a command prompt, except to fulfill documented and validated mission requirements.
+rhel_07_040160_console_keepalive_interval: 28800
```

Comment:
There is already a var to control this setting in `defaults/main.yml` at https://github.com/sscpac/RHEL7-STIG/blob/8819962f2d44334a876148944419166d98a73240/defaults/main.yml#L442-L444

Should override this setting in your group/host vars as needed.